custom_components/qss/const.py

```diff
@@ -14,7 +14,7 @@
 CONF_AUTH_Y_KEY = "y_key"
 
 RETRY_WAIT_SECONDS = 5
-RETRY_ATTEMPTS = 10
+RETRY_ATTEMPTS = 10000
```

Comment:
I don't know if I like this approach, I'll have to think about it again. 
But I think this change should be done independently with the changes necessary for the questdb client update.

Reply:
Yes I can roll it back. But nothing wrong in retrying forever with 5 sec intervals. Much better than retrying for 50 sec and die, if your QestDB server takes longer to reboot, for example:)